client/cmd/root.go

```diff
@@ -56,6 +57,7 @@ func init() {
 	}
 	rootCmd.PersistentFlags().StringVar(&daemonAddr, "daemon-addr", defaultDaemonAddr, "Daemon service address to serve CLI requests [unix|tcp]://[path|host:port]")
 	rootCmd.PersistentFlags().StringVar(&managementURL, "management-url", "", fmt.Sprintf("Management Service URL [http|https]://[host]:[port] (default \"%s\")", internal.ManagementURLDefault().String()))
+	rootCmd.PersistentFlags().StringVar(&adminURL, "admin-url", "https://app.wiretrustee.com", "Admin Panel URL [http|https]://[host]:[port]")
```

Comment:
Default should be https://app.netbird.io

Reply:
Fixed